### PR TITLE
feat: multipart copy support

### PR DESCRIPTION
**Pull request by binghaiwang**
1.multipart copy support
2.add listPart, initMultipart, uploadPart, completeMultipart api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1513,7 +1518,157 @@ var result = yield store.getACL('ossdemo.txt');
...
+```
+
+### .uploadPart(name, uploadId, partNo, data[, options])
+After initiating a Multipart Upload event, you can upload data in parts based on the specified object name and Upload ID. 
+
+parameters:
+
+- name {String} object name
+- uploadId {String} get by initMultipartUpload api
+- partNo (Integer) range is 1-10000, If this range is exceeded, OSS returns the InvalidArgument's error code.
```
Comment (PeterRao) [inline README.md]: () 改成 {}

File: README.md [hunk 2]
```diff
@@ -1513,7 +1518,157 @@ var result = yield store.getACL('ossdemo.txt');
...
+```
+
+### .completeMultipartUpload(name, uploadId, parts[, options])
+After uploading all data parts, you must call the Complete Multipart Upload API to complete Multipart Upload for the entire file.
+
+parameters:
+
+- name {String} object name
+- uploadId {String} get by initMultipartUpload api
+- parts (Array)
```
Comment (PeterRao) [inline README.md]: () 改成 {}

File: README.md [hunk 3]
```diff
@@ -1513,7 +1518,157 @@ var result = yield store.getACL('ossdemo.txt');
...
+  
+  //complete
+  var completeData = yield store.completeMultipartUpload(name, result.uploadId, done);
+  console.log(completeData);
+```
+
+
+### .multipartUpload*(name, file[, options])
 
 Upload file with [OSS multipart][oss-multipart].
```
Comment (PeterRao) [inline README.md]: 注释下封装了哪些接口

**Comment by PeterRao** [comment]
uploadPart接口入参及文档不明确，需要重写

File: README.md (hunk 1 above)
Comment (binghaiwang) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (binghaiwang) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (binghaiwang) [reply README.md]: fixed

**Pull request merged**